src/SpecDataHolder.t.sol

```diff
@@ -136,6 +136,58 @@ contract SpecDataHolderTest is Test {
     specDataHolderWrappedProxyV1.setBadgesAddress(randomAddress);
   }
 
+  function testSetBadgesToRaftsShouldFailForUnauthorizedCaller() public {
```

Comment:
Are we missing happy path tests here?

Reply:
in `setBadgesToRafts` .. I cant actually check `_badgeToRaft` because that variable is private
`mapping(uint256 => uint256) private _badgeToRaft;`

there's not really a happy path for testing to see if `setBadgesToRafts` is properly setting `_badgeToRaft`
you cant query `_badgeToRaft` from the test suite. 

To write a better test we'd need a getter, which I'm not convinced is worth adding to the contract just for tests (on a function we will probably almost never use)

Reply:
Makes sense. But I think this probably is a good reason for us to have getters for these setters in general. Right?